test/unittest/CMakeLists.txt

```diff
@@ -80,14 +72,17 @@ add_test(NAME unittest
 
 if(NOT MSVC)
     # Not running SIMD.* unit test cases for Valgrind
-    add_test(NAME valgrind_unittest
-        COMMAND valgrind --suppressions=${CMAKE_SOURCE_DIR}/test/valgrind.supp --leak-check=full --error-exitcode=1 ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/unittest --gtest_filter=-SIMD.*
-        WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}/bin)
+    find_program(VALGRIND_FOUND valgrind)
```

Comment:
Patch with actually correct usage of `find_program` is already in #1808.
So:
* Variable specified in first argument of `find_program` is name of executable, so this variable should be named `VALGRIND_EXECUTABLE` or `VALGRIND_EXE` or `VALGRIND`.
* Check for Valgrind should be outside `NOT MSVC` branch. If Valgrind ever gains Windows/MSVC support, then `valgrind_unittest` should be run there.

Reply:
ok, done